frontend/src/scenes/notebooks/Notebook/Notebook.tsx

```diff
@@ -102,6 +109,7 @@ export function Notebook({ shortId, editable = false }: NotebookProps): JSX.Elem
 
                         return ''
                     }}
+                    viewMode={cachedNotebook ? NotebookMode.Share : undefined}
```

Comment:
Am I right in thinking that we are solely relying on `cachedNotebook` as a means of asserting that the notebook is being shared? If so, that would mean that the `mode` would remain as the default `NotebookMode.View` which would be confusing.

I guess there is somewhat of a conflation here between being in a mode to setup sharing versus actually viewing a shared notebook.

Maybe it is worth having two separate states relating to sharing on `NotebookMode` for that reason, and setting the mode as a default within Kea when setting up the logic [above](https://github.com/PostHog/posthog/pull/16415/files?diff=unified&w=1#diff-4dfe7cc3e9ce5ec76a83329ec6b525d2fce4b9e2b708c50b1daa8a0b15d93c50R26)

Reply:
Ah, see, that's why I don't like big PRs... totally missed I was using one mode for two totally different things. Great spot